examples/docker/README.md

```diff
@@ -0,0 +1,29 @@
+# Run Remix With docker-compose
+
+very simple example to run remix app with docker and docker-compose
```

Comment:
Could you explain here why someone would want to do this?

Reply:
ok i will add:

**With this example Remix can be run in any server have docker.**